script-templates/OpenTelemetry.DotNet.Auto.psm1.template

```diff
@@ -17,11 +17,26 @@
 #Requires -RunAsAdministrator
 
 $ServiceLocatorVariable = "OTEL_DOTNET_AUTO_INSTALL_DIR"
+$ServiceVersionVariable = "OTEL_DOTNET_AUTO_INSTALL_VERSION"
 
 function Get-Current-InstallDir() {
     return [System.Environment]::GetEnvironmentVariable($ServiceLocatorVariable, [System.EnvironmentVariableTarget]::Machine)
 }
 
+function Get-Current-InstallVersion() {
+    return [System.Environment]::GetEnvironmentVariable($ServiceVersionVariable, [System.EnvironmentVariableTarget]::Machine)
```

Comment:
Should we inspect one of the installed dlls to determine which version is installed instead of relying on an environment variable?

Reply:
That's actually interesting idea, I'll check if I can get right version out of the main dll.

Reply:
Makes it so much better > 
https://github.com/open-telemetry/opentelemetry-dotnet-instrumentation/pull/3042/commits/9411ad8f2003b7d71dc85b1340563a33414a6df6